addons/html_editor/static/tests/table/selection.test.js

```diff
@@ -1658,7 +1785,7 @@ describe("single cell selection", () => {
 
         const BORDER_SENSITIVITY = 5;
         const firstTd = el.querySelector("td");
-        const offset = BORDER_SENSITIVITY + 1;
+        const offset = BORDER_SENSITIVITY + 2;
```

Comment:
To check: there's a bottom margin of -1px on the placeholder so the td shouldn't have moved. Is this then correct?